### PR TITLE
tests/muinstaller-real: test with UBUNTU-SEED fs label

### PR DESCRIPTION
**Pull request by alfonsosanchezbeato**
This goes on top of https://github.com/snapcore/snapd/pull/12753

### CODE REVIEW

File: tests/nested/manual/muinstaller-real/task.yaml [hunk 1]
```diff
@@ -93,7 +95,7 @@ execute: |
   # we don't need the fakestore anymore
   "$TESTSTOOLS"/store-state teardown-fake-store "$STORE_DIR"
 
-  # build the fake-installer snap
+  # build the muinstaller snap
```
Comment (mvo5) [inline tests/nested/manual/muinstaller-real/task.yaml:98]: Thank you!

**Review by mvo5 — APPROVED** [review]
Thank you

File: tests/nested/manual/muinstaller-real/task.yaml [hunk 2]
```diff
@@ -241,9 +249,18 @@ execute: |
...
+      # Order of assertion acking is important
+      for p in "$STORE_DIR"/asserts/*.account-key "$STORE_DIR"/asserts/*.snap-declaration "$STORE_DIR"/asserts/*.snap-revision; do
+          remote.push "$p" asserts/
+          remote.exec sudo snap ack asserts/"$(basename "$p")"
+      done
+      remote.push "$STORE_DIR"/"$snap_name"_*.snap
+
       boot_id=$(tests.nested boot-id)
-      REMOTE_CHG_ID=$(remote.exec sudo snap refresh --amend --no-wait --channel="$version/edge" "$snap_name")
+      REMOTE_CHG_ID=$(remote.exec "sudo snap install --no-wait ${snap_name}_*.snap")
```
Comment (pedronis) [inline tests/nested/manual/muinstaller-real/task.yaml:263]: this probably needs a comment why we are using a local snap install with assertions, we might not need the fakestore if we do this as we could just push and ack the assertions instead?

**Review by pedronis — APPROVED** [review]
thx, one small remark

**Pull request merged**